darts/models/forecasting/prophet_model.py

```diff
@@ -265,6 +297,40 @@ def add_seasonality(
         }
         self._store_add_seasonality_call(seasonality_call=function_call)
 
+    def set_capacity(
+        self,
+        cap: Union[
+            float, Callable[[Union[pd.DatetimeIndex, pd.RangeIndex]], Sequence[float]]
+        ],
+        floor: Union[
+            float, Callable[[Union[pd.DatetimeIndex, pd.RangeIndex]], Sequence[float]]
+        ] = 0,
+    ) -> None:
+        """Set carrying capacities for predicting with logistic growth.
+        These capacities are only used when `Prophet` was instantiated with `growth = 'logistic'`
+        See <https://facebook.github.io/prophet/docs/saturating_forecasts.html> for more information
+        on logistic forecasts.
+
+        The `cap` and `floor` parameters may be:
```

Comment:
You should add a blank line for the bullet-point list to render correctly in HTML (you can try building & rendering the doc with `./gradlew buildDocs` to check the HTML files directly).